webapp-django/crashstats/crashstats/models.py

```diff
@@ -44,39 +44,23 @@
 
 
 class GraphicsDeviceManager(models.Manager):
-    def get_pair(self, adapter_hex, vendor_hex):
-        """Returns (adapter_name, vendor_name) or None"""
+    def get_pair(self, vendor_hex, adapter_hex):
```

Comment:
I flipped the order of these arguments and the ones in `.get_pairs()` because there's a hierarchy to vendor and adapter and it makes more sense to me to do vendor first. Among other things, I keep putting vendor first and then have to correct it later. Now I don't have to do that anymore.